tests/typechecker/control_flow_while_1.jakt

```diff
@@ -0,0 +1,14 @@
+/// Expect:
+/// - output: "PASS\n"
+
+function test(anon mut argument: i64) -> String {
+	while argument > 0 {
```

Comment:
lgtm, but could you make these use spaces instead of hard tabs?

Reply:
vscode had a "from now on, i will use tabs instead of spaces" incident, should be fixed now